notify-github.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# MOVE THIS FILE TO  https://github.com/libero/scripts/ once tested
+set -e
+
+unique_id=$(uuidgen)
+temp_file="github-commit-status-${unique_id}.log"
+
+repository="libero/reviewer"
+# Travis CI builds a merge commit on pull requests, so
+# we want to refer to the second parent of this merge commit
+commit=$(git rev-parse HEAD^2)
+status="success"
+context="travis"
```

Comment:
there should be some validation on the variables passed in being present, otherwise the script should `exit 1`

Reply:
`context` could be a more descriptive value

Reply:
> `context` could be a more descriptive value

suggestions?

Reply:
`storybook` as it should be future-proof to changing the CI tool if we go down that route

Reply:
(e.g. you can make the branch protected and stop PRs from being merged if that check is not there, etc.)